exercises/space-age/src/example.h

```diff
@@ -1,18 +1,18 @@
 #ifndef SPACE_AGE_H
 #define SPACE_AGE_H
 
-/* Number of seconds in a Earth year */
-#define EARTH 31557600
+typedef enum planets {
+   mercury = 0,
+   venus,
+   earth,
+   mars,
+   jupiter,
+   saturn,
+   uranus,
+   neptune,
+   max_planet = neptune
```

Comment:
It was my understanding that this is good usage as it prevents having to reference `neptune` directly in any limit case (e.g. a `for` loop). If for example, pluto is added, then in this manner only the `enum` needs changing. Sure it needs changing in two places (right next to each other), but this beats having to change it in the enum _and_ any limit cases throughout the code. Is this not correct?